hack/update-license-header.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
```

Comment:
this is included in `Makefile`

Reply:
CI is suitable for inspection, because the CI phase replacement does not affect the user's code, and the update of the code is handed over to the user.

Reply:
if a user sees there is an error in CI report, he/she will re-run `make lint-license` and push the code again.

Reply:
can CI report errors directly in license-header-checker instead of relying on git diff?

Reply:
no, but it will show which file got replaced.